src/main/java/com/crowdin/action/UploadFromContextAction.java

```diff
@@ -23,34 +21,27 @@
     public void performInBackground(AnActionEvent anActionEvent) {
 
         final VirtualFile file = CommonDataKeys.VIRTUAL_FILE.getData(anActionEvent.getDataContext());
-        Crowdin crowdin = new Crowdin(anActionEvent.getProject());
         Project project = anActionEvent.getProject();
-        String branch = GitUtil.getCurrentBranch(project);
-        crowdin.uploadFile(file, branch);
+        CrowdinProperties properties = CrowdinPropertiesLoader.load(project);
+        Crowdin crowdin = new Crowdin(project, properties.getProjectId(), properties.getApiToken(), properties.getBaseUrl());
+        String branch = properties.isDisabledBranches() ? "" : GitUtil.getCurrentBranch(project);
+        crowdin.uploadFile(file, "/values-%android_code%/%original_file_name%", branch);
```

Comment:
`/values-%android_code%/%original_file_name%` move to constant. Probably we have the same constant in listener